solidity/contracts/hooks/OPStackHook.sol

```diff
@@ -75,7 +75,7 @@ contract OPStackHook is AbstractMessageIdAuthHook {
     ) internal override {
         bytes memory payload = abi.encodeCall(
             AbstractMessageIdAuthorizedIsm.verifyMessageId,
-            message.id()
+            (message.id(), metadata.msgValue(0))
```

Comment:
any reason for this not to be on the base contract?

Reply:
The call to the external bridge doesn't otherwise have access to `metadata.msgValue(0)` so I changed the _sendMessageId signature.